cmd/promtool/main.go

```diff
@@ -14,18 +14,123 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"strconv"
 	"strings"
+	"time"
 
+	"golang.org/x/net/context"
+
+	"github.com/prometheus/client_golang/api/prometheus"
 	"github.com/prometheus/common/version"
 	"github.com/prometheus/prometheus/config"
 	"github.com/prometheus/prometheus/promql"
 	"github.com/prometheus/prometheus/util/cli"
 )
 
+var (
+	server  = flag.String("server", "http://localhost:9090", "URL of the Prometheus server to query (default http://localhost:9090)")
```

Comment:
Are you sure it's necessary to include the default value again? The flag usage printer should do that automatically.

I'm not a big fan of global variables, it makes it harder to test and easier to leave flags in place when removing the code using them. But most of all, this will add the two flags to all promtool commands, even though only two are using them.